src/main/java/decodes/cwms/CwmsTimeSeriesDAO.java

```diff
@@ -1489,7 +1489,8 @@ else if (!TextUtil.strEqualIgnoreCase(recUnitsAbbr, ctsUnitsAbbr))
         }
         if (rrhandle != null)
         {
-            rrhandle.addRecNum(rec.getRecordNum());
+            // TODO: implement new Scheme
+            //rrhandle.addRecNum(rec.getRecordNum());
```

Comment:
```suggestion
            //rrhandle.addRecNum(rec.getRecordNum());
            throw new UnsupportedOperationException("rrhandle.addRecNum(rec.getRecordNum())");
```

Reply:
I should have been clearer in the message, these need to be replace *before* this gets merged in. The entire computation process system fails without these records. I just wanted to allow for initial review before adding that much code.